spec/models/manageiq/providers/cloud_manager/vm_spec.rb

```diff
@@ -44,4 +48,158 @@
       end
     end
   end
+
+  context "queued methods" do
+    before do
+      allow(ems).to receive(:queue_name_for_ems_operations).and_return(queue_name)
```

Comment:
This shouldn't be needed

Reply:
True, but it's not a bad thing to do since it verifies that it's using the queue name we expect.

Reply:
Oh we should still verify that it is using the queue name we expect, I just don't think we should stub this since it could hide issues later on

Reply:
I guess I'm not following what you would prefer instead, since there's currently no setter for it. Suggestions?

Reply:
What happens if you don't have this stub at all?

Reply:
It just defaults to 'generic'. I thought it would be good to use a non-default though.

Reply:
@agrare oops, actually it's nil.

Reply:
So now that https://github.com/ManageIQ/manageiq/pull/19627 is in we don't need this right?